third_party/move/tools/move-smith/src/names.rs

```diff
@@ -0,0 +1,162 @@
+use arbitrary::Arbitrary;
+use std::collections::HashMap;
+
+// #[derive(Debug, Clone, PartialEq, Eq, Hash)]
+// pub struct Identifier {
+//     pub name: String,
+// }
+
+pub type Identifier = String;
```

Comment:
Could we use newtype pattern here instead (https://doc.rust-lang.org/rust-by-example/generics/new_types.html) to get additional type safety benefits instead of just type name aliasing?

Reply:
Done